Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -64,6 +64,24 @@ extension Event {
 
       /// A type describing data tracked on a per-test basis.
       struct TestData {
+        /// A lightweight struct containing information about a single issue.
+        struct IssueInfo: Sendable {
+          /// The source location where the issue occurred.
+          var sourceLocation: SourceLocation?
+          
+          /// A detailed description of what failed (using expanded debug description).
+          var description: String
+          
+          /// Whether this issue is a known issue.
+          var isKnown: Bool
+          
+          /// The test's display name, if any.
+          var testDisplayName: String?
+          
+          /// The test case arguments, formatted for display (for parameterized tests).
+          var testCaseArguments: String?
```

Comment:
We should store these two properties on the outer `TestData` struct rather than in `IssueInfo`, since they will be the same for each test and having them here means they would be repeated multiple times for tests which record multiple issues